CHANGES.txt

```diff
@@ -1,5 +1,6 @@
 1.0.0
 -----
+ * Integrate delta-coverage-plugin plugin (CASSSIDECAR-163)
```

Comment:
I don't think we need to add this change here because it's not anything that touches code

Reply:
I'll remove it then :-)